### PR TITLE
Clarifying overutilization requirement

### PR DESCRIPTION
**Pull request by bergerhoffer**

### CODE REVIEW

**Review by damemi — APPROVED** [review]
/lgtm
Thanks!

**Comment by bergerhoffer** [comment]
Hi @kasturinarra, can you take a look at this wording adjustment, where a node's usage doesn't have to be above **all** parameters to be considered overutilized. Thanks!

Preview: https://adjust-overutilization-wording--ocpdocs.netlify.app/openshift-enterprise/latest/nodes/scheduling/nodes-descheduler.html#nodes-descheduler-strategies_nodes-descheduler

**Comment by kasturinarra** [comment]
> Hi @kasturinarra, can you take a look at this wording adjustment, where a node's usage doesn't have to be above **all** parameters to be considered overutilized. Thanks!
> 
> Preview: https://adjust-overutilization-wording--ocpdocs.netlify.app/openshift-enterprise/latest/nodes/scheduling/nodes-descheduler.html#nodes-descheduler-strategies_nodes-descheduler

Reviewed and added my comments, thanks !!

File: modules/nodes-descheduler-strategies.adoc [hunk 1]
```diff
@@ -12,7 +12,7 @@ The `LowNodeUtilization` strategy finds nodes that are underutilized and evicts
 +
 The underutilization of nodes is determined by several configurable threshold parameters: CPU, memory, and number of Pods. If a node's usage is below the configured thresholds for all parameters (CPU, memory, and number of Pods), then the node is considered to be underutilized.
```
Comment (lbarbeevargas) [inline modules/nodes-descheduler-strategies.adoc:13]: @bergerhoffer Just want to double check that the same language doesn't need to be updated for _underutilization_ as well.

**Review by lbarbeevargas — COMMENTED** [review]
Quick question to confirm that the same language update doesn't need to be applied to _underutilization_ as well. Otherwise, LGTM.

File: modules/nodes-descheduler-strategies.adoc (hunk 1 above)
Comment (kasturinarra) [reply modules/nodes-descheduler-strategies.adoc:13]: I think it should be updated as well @lbarbeevargas .

@damemi what do you think ?

Comment (bergerhoffer) [reply modules/nodes-descheduler-strategies.adoc:13]: Good question - @lbarbeevargas @kasturinarra according to the slack discussion that started this, all of the threshold parameters need to be met to qualify for underutilization, but only one of them needs to qualify for overutilization. @damemi can correct me if I'm wrong though

Comment (damemi) [reply modules/nodes-descheduler-strategies.adoc:13]: This is correct (verified by [these comments in the descheduler code](https://github.com/kubernetes-sigs/descheduler/blob/master/pkg/descheduler/strategies/lownodeutilization.go#L392-L415)).

**Pull request merged**